src/chrome/content/rules/Sigxcpu.org.xml

```diff
@@ -1,40 +1,29 @@
 <!--
-	Nonfunctional subdomains:
+	Non-functional hosts
+		Couldn't connect to server:
+			 - gluon.sigxcpu.org
+			 - honk6.sigxcpu.org
```

Comment:
I got the following results using `curl` and I do have a ipv6 compatible network. How should I comment the site's behavior?
```
$ curl -v https://gluon.sigxcpu.org
* Rebuilt URL to: https://gluon.sigxcpu.org/
*   Trying 95.142.175.121...
* TCP_NODELAY set
* connect to 95.142.175.121 port 443 failed: Connection refused
*   Trying 2001:4b98:dc0:51:216:3eff:fe5f:8531...
* TCP_NODELAY set
* Immediate connect fail for 2001:4b98:dc0:51:216:3eff:fe5f:8531: Network is unreachable
*   Trying 2001:4b98:dc0:51:216:3eff:fe5f:8531...
* TCP_NODELAY set
* Immediate connect fail for 2001:4b98:dc0:51:216:3eff:fe5f:8531: Network is unreachable
* Failed to connect to gluon.sigxcpu.org port 443: Connection refused
* Closing connection 0
curl: (7) Failed to connect to gluon.sigxcpu.org port 443: Connection refused
```
```
$ curl -v https://honk6.sigxcpu.org
* Rebuilt URL to: https://honk6.sigxcpu.org/
*   Trying 2001:7b8:3de:2011::3...
* TCP_NODELAY set
* Immediate connect fail for 2001:7b8:3de:2011::3: Network is unreachable
* Closing connection 0
curl: (7) Couldn't connect to server

```